libcnb-test/src/container_context.rs

```diff
@@ -78,12 +80,101 @@ impl<'a> PrepareContainerContext<'a> {
         self
     }
 
-    /// Creates and starts the container configured by this context.
+    /// Creates and starts the container configured by this context with the image's default
+    /// CNB process.
+    ///
```

Comment:
I wonder if it would be worth mentioning here (and in the docs for the related methods below) how container teardown is handled?

Reply:
I don't have a strong opinion on this other than this is IMO out of scope for this PR since the way we do teardown didn't change. If we want this as RustDoc, we can do a separate PR or issue. :)

Reply:
Ah true. I'll file an issue for it. However I'd love for us to start making progress though the long tail of follow-up issues, and not just the big feature PRs at some point too :-)

Reply:
> and not just the big feature PRs at some point too :-)

I guess that's a matter of priorities. Missing features block our progress with the actual buildpacks, doc improvements do not (granted that better docs would reduce friction). :-) But I think we're close to the point where the actual blockers are gone. :)